src/nanorc/__main_timing__.py

```diff
@@ -57,7 +58,7 @@ def timingcli(ctx, obj, traceback, loglevel, log_path, cfg_dumpdir, kerberos, ti
 
     obj.console.print(Panel.fit(grid))
 
-    port_offset = 500 + partition_number * 1_000
+    port_offset = -1100 + partition_number * 500
```

Comment:
-1100? Are you _sure_?

Reply:
Well, since the configuration is _always_ assigning port 3333 as the first port, indeed I'm sure. With https://github.com/DUNE-DAQ/daqconf/pull/89/commits/a7b2c7c76322cb9884c693e4bc5c3823c635fa40, this line becomes completely useless and goofy.